dockerfiles/base.dockerfile

```diff
@@ -14,3 +14,4 @@ ENTRYPOINT [ "/entrypoint.sh" ]
 CMD ["tail", "-f", "/dev/null"]
 
 LABEL "che.base.image" ${BASE_IMAGE}
+
```

Comment:
do we need this change ?

Reply:
we do this when changes in `base.dockerfile`, `entrypoint,sh` or in `install-editor-tooling.sh`

Reply:
action [build-sidecar-images-on-push.yml](https://github.com/eclipse-che/che-devfile-registry/blob/main/.github/workflows/build-sidecar-images-on-push.yml) runs [build-sidecar-images-on-push.sh](https://github.com/eclipse-che/che-devfile-registry/blob/main/build/workflows/build-sidecar-images-on-push.sh) which detects the changes in the last commit and decides whether it needs to rebuild only specific image or all the images.

Reply:
If the latest commit does not change the dockerfile(s) or the common part, we don't need to rebuild anything.
That's why I added an empty line here.

Reply:
yes but the github action could be a parametrised action like
https://github.com/eclipse-che/che-theia/blob/main/.github/workflows/check-theia-branch.yml#L13-L22

so script would receive as well arguments coming from the action so we don't need to commit files to trigger again the job (like if we need to respin anytime a full build)

Reply:
My latest commit (this fixup) will not contain anything related to the dockerfiles, and anything will be built.
To be able to build the images manually we need to add another gh-action. But do we need it?

Reply:
so instead of having 
```
if [ -n "${CHANGES}" ]; then
  echo -e "\nRebuild ALL images"
```

we check as well if there is parameter saying to rebuild anyway all the images

and in the github .yaml we provide the parameter to the build.sh script (that may be empty, default case)

Reply:
> so instead of having
> 
> ```
> if [ -n "${CHANGES}" ]; then
>   echo -e "\nRebuild ALL images"
> ```
> 
> we check as well if there is parameter saying to rebuild anyway all the images

ok, trying to add it.